deal.II-toolchain/packages/trilinos.package

```diff
@@ -127,11 +130,14 @@ if [ ! -z "${SUPERLU_DIR}" ]; then
     CONFOPTS="\
         ${CONFOPTS} \
           -D TPL_ENABLE_SuperLUDist:BOOL=ON \
-          -D Teuchos_ENABLE_COMPLEX:BOOL=OFF \
           -D TPL_SuperLUDist_LIBRARIES:FILEPATH=${SUPERLU_DIR}/lib/libsuperlu_dist.${LDSUFFIX} \
           -D SuperLUDist_INCLUDE_DIRS:PATH=${SUPERLU_DIR}/include \
           -D HAVE_SUPERLUDIST_LUSTRUCTINIT_2ARG:BOOL=ON \
           -D HAVE_SUPERLUDIST_ENUM_NAMESPACE:BOOL=ON"
+
+#          -D Teuchos_ENABLE_COMPLEX:BOOL=OFF \
```

Comment:
Maybe just remove this line, I am not sure what would happen if someone tried to uncomment it.